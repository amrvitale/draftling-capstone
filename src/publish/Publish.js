import React from 'react'
import { Link } from 'react-router-dom'
import DraftlingPage from '../draftlingpage/DraftlingPage'
import ApiContext from '../ApiContext'

class Publish extends React.Component {
    render() {
        return (
            <div>

            </div>
        );
    }
}

export default Publish;