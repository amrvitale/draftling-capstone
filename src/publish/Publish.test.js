import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Publish from './Publish'

describe (`Publish component`, () => {
    it('renders a .Publish by default', () => {
        const wrapper = shallow(<Publish />)
        expect (toJson(wrapper)).toMatchSnapshot()
    })

    
})