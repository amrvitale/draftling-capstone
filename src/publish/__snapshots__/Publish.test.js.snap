// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Publish component renders a .Publish by default 1`] = `<div />`;
